.circleci/config.yml

```diff
@@ -44,6 +44,44 @@ jobs:
       - store_artifacts:
           path: build/reports/tests
 
+  publish_data:
+    executor: hmpps/java
+    docker:
+      - image: cimg/openjdk:11.0
+        environment:
+          POSTGRES_DB: circle_test
+      - image: circleci/postgres:10-alpine-ram
+        environment:
+          POSTGRES_PASSWORD: password
+    steps:
+      - checkout
+      - hmpps/create_app_version
+      - run:
+          name: Migrate the database
+          command: |
+            wget -qO- https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/7.7.0/flyway-commandline-7.7.0-linux-x64.tar.gz | \
+              tar xvz && sudo ln -s `pwd`/flyway-7.7.0/flyway /usr/local/bin
+            flyway migrate -url="jdbc:postgresql://localhost:5432/$POSTGRES_DB" -user="postgres" -password="password" -locations="src/main/resources/db/migration"
+      - run:
+          name: Build data dictionary
+          command: |
+            sudo apt update && sudo apt install graphviz
+            script/build-data-dictionary.sh
+      - setup_remote_docker
```

Comment:
could we use the build-docker job provided by the hmpps orb for these subsequent tasks?